docs/source/about/advanced_usage.rst

```diff
@@ -616,9 +616,58 @@ keybindings
          You need to use the id value inside the parenthesis, e.g. ``3``.
 
       **Windows**
-         .. code-block:: batch
+         During Sunshine startup, you should see the list of detected displays:
 
-            tools\dxgi-info.exe
+         .. code-block:: text
```

Comment:
```suggestion
         .. code-block:: doscon
```

What do you think? https://pygments.org/docs/lexers/#pygments.lexers.shell.MSDOSSessionLexer

Reply:
I don't think it fits in this case anymore since we are now providing log output 🤔